client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/WatcherDocumentationIT.java

```diff
@@ -132,4 +135,59 @@ public void onFailure(Exception e) {
         }
     }
 
+    public void testActivateWatch() throws Exception {
```

Comment:
We will have a new ay to deal with the docs+test, we should wait to merge until it is merged.

Reply:
@hub-cap which is the issue that it is introducing this change?

Reply:
#34125 and it looks like its merged!